### DISCUSSION TITLE
Using mailable when testing with mail::fake()

### DISCUSSION THREAD

**Discussion by lloricode** [spatie/ray#478] [Q&A] [answered]
Is there a possible way to use `ray()->mailable();` properly without removing `Mail:fake()`?

I got this result when running in testing, then removing fake will send this actual message
![Screenshot from 2021-06-21 08-41-14](https://user-images.githubusercontent.com/8251344/122693654-407af400-d26d-11eb-8443-cd3622e82365.png)

thank you

**Comment by freekmurze** [comment, marked as answer]
This has been fixed in the latest release of `laravel-ray`.

**Reply by lloricode** [reply]
thanks, will check this
